ragna/deploy/_ui/components/file_uploader.py

```diff
@@ -115,15 +115,20 @@ def perform_upload(self, event=None, after_upload_callback=None):
                 .fileListContainer {
                     display:flex;
                     flex-direction: row;
-                    height: 44px;
-                    overflow:scroll;
-                    padding-top:14px;
+                    flex-wrap: wrap;
+                    height: 40px;
+                    overflow-y:scroll;
+                    overflow-x:hidden;
+                    margin-top: 10px;
+                    margin-bottom: 10px;
+                    padding-top:5px;
                     padding-left:6px;
+                    padding-bottom:5px;
                 }
                 
                 .chat_document_pill {
                     background-color: rgb(241,241,241);
-                    margin-top:0px; 
+                    margin-top:5px; 
```

Comment:
To make things a bit more readable when many pills, introduce a small amount of white space between them.